client/my-sites/themes/theme-showcase.jsx

```diff
@@ -155,15 +134,41 @@ class ThemeShowcase extends Component {
 			( this.props.isJetpackSite && ! this.props.isAtomicSite ) ||
 			( this.props.isAtomicSite && this.props.siteCanInstallThemes );
 
+		const staticFilters = this.getStaticFilters();
 		return {
 			...( shouldShowMyThemesFilter && {
-				MYTHEMES: this.staticFilters.MYTHEMES,
+				MYTHEMES: staticFilters.MYTHEMES,
 			} ),
-			ALL: this.staticFilters.ALL,
+			ALL: staticFilters.ALL,
 			...this.subjectFilters,
 		};
 	};
 
+	getStaticFilters = () => {
+		// As the values of these filter is static and won't be changed, we use it to check a filter
```

Comment:
We can then remove this comment; it's not correct anymore

Reply:
That makes sense, thanks, I've removed the comment!